library/core/src/slice/ascii.rs

```diff
@@ -114,18 +114,19 @@ impl [u8] {
     /// Returns a byte slice with leading ASCII whitespace bytes removed.
     ///
     /// 'Whitespace' refers to the definition used by
-    /// `u8::is_ascii_whitespace`.
+    /// [`u8::is_ascii_whitespace`].
+    ///
+    /// [`u8::is_ascii_whitespace`]: u8::is_ascii_whitespace
```

Comment:
? is this explicit doc link actually required? it's my experience that this sort of thing actually links by itself nowadays as long as the `[]` are used. certainly this part is linked, right?

https://github.com/rust-lang/rust/blob/69ae958311d65fb71d3db02c28f8aabd207ff759/library/core/src/slice/ascii.rs#L208

Reply:
Looks like it's not required, I guess I still do that out of caution/habit.
I can remove the explicit links.

Reply:
Removed in the latest force push at `9fb49fa`.

Reply:
it's an understandable tic, yeah.